flang/test/Semantics/OpenMP/workshare02.f90

```diff
@@ -61,6 +69,16 @@ subroutine workshare(aa, bb, cc, dd, ee, ff, n)
   j = j - my_func()
   !$omp end atomic
 
+  !ERROR: User defined IMPURE non-ELEMENTAL function 'impure_my_func' is not allowed in a WORKSHARE construct
```

Comment:
Should there be a comma?

Reply:
Between the `IMPURE` and `non-ElEMENTAL`?
```
User defined IMPURE, non-ELEMENTAL function
```